blackbox/docs/release_notes/2.0.4.txt

```diff
@@ -0,0 +1,63 @@
+.. _version_2.0.4:
+
+=============
+Version 2.0.4
+=============
+
+Released on 2017/07/06.
+
+.. NOTE::
+
+   If you are upgrading a cluster, you must be running CrateDB
+   :ref:`version_1.1.3` or higher before you upgrade to 2.0.4.
+
+   If you want to perform a :ref:`rolling_upgrade`, your current CrateDB
+   version number must be :ref:`version_2.0.0`. If you want to
+   upgrade from a version prior to this, the upgrade will introduce all of the
+   breaking changes listed for :ref:`version_2.0.0`, and will require a
+   :ref:`full_restart_upgrade`.
+
+   Consult the :ref:`version_2.0.0_upgrade_notes` for ``2.0.0`` when
+   upgrading.
+
+.. WARNING::
+
+   If you upgrade from a version greater or equal than 2.0.0 you need to
+   perform a :ref:`rolling_upgrade` in order to prevent BLOB data loss caused
+   by a bug introduced with :ref:`version_2.0.0`.
```

Comment:
With replicas > 1 and the cluster always returning to a green cluster state between switching upgrading the different nodes.